### ISSUE TITLE
Keep structure of config.pl on saving changes

### ISSUE THREAD

**Issue by kr4ut** [backuppc/backuppc#275]
First off big thanks for all the efforts ... especially into the revamp with version 4 :+1: 
While working with Ansible templating and etckeeper we noticed an unfortunate behavior.
Basically every small config change via the web UI leads to huge diff on the config.pl like:
```
 $Conf{CgiStatusHilightColor} = {
+  'Reason_no_ping' => '#ffff99',
   'Status_backup_in_progress' => '#66cc99',
   'Reason_backup_canceled_by_user' => '#ff9900',
-  'Disabled_OnlyManualBackups' => '#d1d1d1',
   'Reason_backup_failed' => '#ffcccc',
-  'Reason_no_ping' => '#ffff99',
-  'Reason_backup_done' => '#ccffcc',
-  'Disabled_AllBackupsDisabled' => '#d1d1d1'
+  'Disabled_AllBackupsDisabled' => '#d1d1d1',
+  'Disabled_OnlyManualBackups' => '#d1d1d1',
+  'Reason_backup_done' => '#ccffcc'
 };
```
Obviously this is no change of actual values but just a random order of the items.
It doesn't matter on the actual parsing of the configuration values and keeping them.
But it an unfortunate hustle when working with version control and config management.
@craigbarratt Couldn't this be solved by a simple sorting of the items before writing the conf?

**Comment by craigbarratt** [comment]
I added SortKeys(1) to the Data::Dumper object; commit 341668f.

**Comment by kr4ut** [comment]
@craigbarratt Late but big thanks after also upgrading our production instances. Can confirm this simple and elegant solution works pretty fine now across the fleet :smiley:

**Comment by craigbarratt** [comment]
Thanks for confirming.  This fix was included in the recently released 4.3.1.
